assets/src/edit-story/app/media/test/useUploadMedia.test.js

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
Nit: No need for the `.test` suffix in the file name

Reply:
That came from me. I'm ok to drop that pattern if others are feeling strongly.

Reply:
Done.